pkg/upgrade/upgrade.go

```diff
@@ -171,6 +184,10 @@ func (u *Upgrade) Run(ctx context.Context) error {
 
 	u.l.Infof("Everest has been upgraded to version %s", upgradeEverestTo)
 
+	if err := u.ensureEverestAccountsIfNotExists(ctx); err != nil {
+		return err
+	}
```

Comment:
Should we do this conditionally? The user may use this CLI version to upgrade from 0.9 to 0.10 and in that case we don't want to run this code. We only want to run it if upgrading to 0.11.